src/IB/IBFEMethod.cpp

```diff
@@ -3638,24 +2872,32 @@ IBFEMethod::commonConstructor(const std::string& object_name,
                                      << std::endl);
         }
         d_fe_family[part] = LAGRANGE;
+        d_fe_family_position[part] = LAGRANGE;
+        d_fe_family_force[part] = LAGRANGE;
+        d_fe_family_pressure[part] = LAGRANGE;
         d_default_quad_type[part] = QGAUSS;
+        d_default_quad_type_stress[part] = QGAUSS;
+        d_default_quad_type_force[part] = QGAUSS;
         if (mesh_has_first_order_elems)
         {
             d_fe_order[part] = FIRST;
+            d_fe_order_position[part] = FIRST;
+            d_fe_order_force[part] = FIRST;
+            d_fe_order_pressure[part] = FIRST;
             d_default_quad_order[part] = THIRD;
+            d_default_quad_order_stress[part] = THIRD;
+            d_default_quad_order_force[part] = THIRD;
         }
         if (mesh_has_second_order_elems)
         {
             d_fe_order[part] = SECOND;
+            d_fe_order_position[part] = SECOND;
+            d_fe_order_force[part] = SECOND;
+            d_fe_order_pressure[part] = SECOND;
             d_default_quad_order[part] = FIFTH;
+            d_default_quad_order_stress[part] = FIFTH;
+            d_default_quad_order_force[part] = FIFTH;
```

Comment:
Shouldn't all of this also be in the base class now?

Reply:
Dropped.